data/json/items/melee/swords_and_blades.json

```diff
@@ -2826,7 +2829,7 @@
   },
   {
     "type": "GENERIC",
-    "//": "Do these even exist?",
+    "//": "Do these even exist? Answer: yeah they did, at least there are two instances from ancient egypt( although this one was more for ceremonies ) and ancient china",
```

Comment:
```suggestion
    "//": "Do these even exist? Answer: yeah they did, at least there are two instances from ancient Egypt( although this one was more for ceremonies ) and ancient China",
```

Reply:
added the shaolin spade as the chinese example